resources/views/pages-legacy/leaderboardinfo.blade.php

```diff
@@ -173,8 +173,6 @@
         // else
         //    echo "<a href='leaderboardinfo.php?i=$lbID&amp;c=$count&amp;f=1'>Show Friends Only</a> - <b>All Results</b><br><br>";
 
-        echo "<div class='larger'>$lbTitle: $lbDescription</div>";
```

Comment:
Went ahead and removed this while I was messing with leaderboards. I noticed this is duplicative with content directly above it on the page.